addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -823,6 +823,18 @@ test("chat window of channels should not have 'Open in Discuss' (mobile)", async
     await contains(".o-dropdown-item", { text: "Open in Discuss", count: 0 });
 });
 
+test("click on an odoo link should fold the chat window (mobile)", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({});
+    patchUiSize({ size: SIZES.SM });
+    await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", `http://${browser.location.host}/odoo.com`);
+    triggerHotkey("control+Enter");
```

Comment:
Post the message with dedicate `.fa-paper-plane` button.
The `control+Enter` should actually not work in mobile!

The test pass because test lacks `test.tags("mobile")`. But even then there's probably a but on runbot which incited skipping these tests: https://github.com/odoo/odoo/pull/202976/commits/15260cfc24679906cecf23acc2fd4914cbb7fc33

Thankfully here the button is there in desktop too so you shouldn't have to skip test even though its mobile environment is not as good as it should.

Reply:
I would move the test to `mobile.test.js` so it has the tag `mobile` (a HOOT change from 2 months ago refuse to have global "desktop" tag and local "mobile" tag...).

Hopefully the test pass with the "mobile" tag, and make sure to replace the ctrl+Enter by the mobile interaction instead, that is click on "send" button.

_(I had to merge a fix while skipping test because of this problem... https://github.com/odoo/odoo/pull/210802, That sucks but it is what it is with HOOT not working properly for mobile tests...)_